drivers/netdev_ieee802154_submac/netdev_ieee802154_submac.c

```diff
@@ -207,12 +204,13 @@ static void submac_tx_done(ieee802154_submac_t *submac, int status,
                            ieee802154_tx_info_t *info)
 {
     (void)status;
-    (void)info;
     netdev_ieee802154_submac_t *netdev_submac = container_of(submac,
                                                              netdev_ieee802154_submac_t,
                                                              submac);
     netdev_t *netdev = (netdev_t *)netdev_submac;
 
+    netdev_submac->retrans = info->retrans;
```

Comment:
Ah yes, indeed this works. But maybe we should handle the case where there are HW retrans but no tx_info in the SubMAC. That way, the SubMAC always gives this information

Reply:
alternatively, we can return an error when `NETOPT_TX_RETRIES_NEEDED` is called in such case

Reply:
Ah I wasn't sure if `info` could be `NULL`.